src/tasks/index.ts

```diff
@@ -772,6 +792,57 @@ export default new class TaskQueue {
     }
   }
 
+  /**
+   * Updates current year approved contracts to SAP
+   *
+   * @param {object} data task data
+   * @param {function} callback task callback
+   */
+  private updateCurrentYearApprovedContractsToSapTask = async (data: any, callback: Queue.ProcessFunctionCb<any>) => {
```

Comment:
Add return type and docs

Reply:
This function performs side effects, so it returns promise of void. The returns inside function body are just to stop the function execution. The returned failTask function returns void, as does the logReject.